### ISSUE TITLE
Difficult to spot the name differences

### ISSUE THREAD

**Issue by marcel1334** [mi-erasmusmc/Tantalus#2]
It took me a while to see what was changed in the two examples underneath. 
---------
0.25 ML TICK-BORN ENCEPHALITIS
VACC 0.0048 MG/ML Injectable
Suspension [Fsme Immun] Box of
10

0.25 ML TICK-BORN
ENCEPHALITIS VACC 0.0048
MG/ML Injectable Suspension
[FSME-Immun] Box of 10

------
rivastigmine 0.554 MG/HR
Transdermal System [Rivastigmin-
Neuraxpharm] Box of 84

rivastigmine 0.554 MG/HR
Transdermal System
[Rivastigmin-Neuraxpharm] Box of
84

--------
I had to copy and past the values in a text editor to find it. Would be great if it is possible to highlight the part in the text that is different? Some thing like in "beyond compare". Or show the new value under the old value like the examples above. In this case you see the difference faster:

0.25 ML TICK-BORN ENCEPHALITIS VACC 0.0048 MG/ML Injectable Suspension [Fsme Immun] Box of 10
0.25 ML TICK-BORN ENCEPHALITIS VACC 0.0048 MG/ML Injectable Suspension [FSME-Immun] Box of 10

rivastigmine 0.554 MG/HR Transdermal System [Rivastigmin-Neuraxpharm] Box of 84
rivastigmine 0.554 MG/HR Transdermal System [Rivastigmin Neuraxpharm] Box of 84

Strange thing is that the linebreaks in the names are not the same for new and old value. This makes finding the difference also pretty tricky. Maybe the columns widths for new/old value are not the same.

**Comment by PRijnbeek** [comment]
Yes, I agree. I am afraid it will become a bit messy to highlight the differences, maybe i can make it an option to switch on an off.

In this case it is the -. You can see in the right most column how many characters are different which may help a bit.

**Comment by PRijnbeek** [comment]
The diffObject package could help here:

<img width="400" alt="screenshot 2018-06-11 16 17 07" src="https://user-images.githubusercontent.com/7252105/41237076-f2a009d8-6d92-11e8-93cc-70bcb93db454.png">
